### ISSUE TITLE
Update UI with some css

### ISSUE THREAD

**Issue by SimonLab** [dwyl/phoenix-liveview-chat-example#4]
At the moment the application is pretty simple:
![image](https://user-images.githubusercontent.com/6057298/143075829-69630f91-91f0-4ad5-8d0b-3c66e2d020a4.png)

- [ ] Add some css to make it nicer to use

**Comment by nelsonic** [comment]
@SimonLab want to try and add UI with Tailwind? https://github.com/dwyl/technology-stack/issues/94

**Comment by nelsonic** [comment]
`lib/liveview_chat_web/templates/layout/root.html.heex`

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8"/>
    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <meta name="csrf-token" content={csrf_token_value()}>
    <%= live_title_tag assigns[:page_title] || "LiveviewChat", suffix: " · Phoenix Framework" %>
    <!-- <link phx-track-static rel="stylesheet" href={Routes.static_path(@conn, "/assets/app.css")}/> -->
    <script defer phx-track-static type="text/javascript" src={Routes.static_path(@conn, "/assets/app.js")}></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/alpinejs/3.10.2/cdn.js" defer></script>
    <script src="https://cdn.tailwindcss.com"></script>
  </head>
  <body class="mx-5 mt-5">
    <header>
      <section class="container">
        <nav>
          <ul class="float-right">
            <%= if @loggedin do %>
              <li>
                <img width="42px" src={@person.picture}/>
              </li>
              <li class="text-blue">
                <%= link "logout", to: "/logout" %>
              </li>
            <% else %>
              <li class="bg-green-600 text-white  p-2 w-full mb-2 font-bold">
                <%= link "Login", to: "/login" %>
              </li>
            <% end %>
          </ul>
        </nav>
        <h1 class="text-5xl mb-4 text-center font-mono">LiveView Chat Example</h1>
      </section>
    </header>
    <%= @inner_content %>
  </body>
</html>
```

`lib/liveview_chat_web/templates/message/messages.html.heex`

```html
<ul id='msg-list' phx-update="append">
   <%= for message <- @messages do %>
     <li id={"msg-#{message.id}"}>
       <b><%= message.name %>:</b>
       <%= message.message %>
     </li>
   <% end %>
</ul>

<.form let={f} for={@changeset} id="form" phx-submit="new_message" phx-hook="Form">

  <%= if @loggedin do %>
    <%= text_input f, :name, id: "name", value: @person.givenName,
    class: "hidden" %>
  <% else %>
    <%= text_input f, :name, id: "name", placeholder: "Your name", autofocus: "true",
     class: "border p-2 w-full mb-2 mt-2" %>
    <%= error_tag f, :name %>
  <% end %>

   <%= text_input f, :message, id: "msg", placeholder: "Your message",
   class: "border p-2 w-full mb-2 mt-2"  %>
   <p class=" text-amber-600">
    <%= error_tag f, :message %>
   </p>
   <%= submit "Send", class: "bg-green-600 text-white px-4 py-2 rounded-xl float-right" %>
 </.form>
```

not adding it to my _current_ PR cause I don't want to change _everything_ in one go.
and if we were doing this properly we would figure out how to have a position fixed for the `<input msg>` and `<submit "Send">` button so that it looks good on mobile too. 

**`#help-wanted`**

**Comment by nelsonic** [comment]
@SimonLab any objections to me including this in my PR #46 ?

**Comment by SimonLab** [comment]
@nelsonic adding the updated UI to the PR sounds good :+1:

**Comment by nelsonic** [comment]
Re-added the Presence, but in the Nav:
![image](https://user-images.githubusercontent.com/194400/174161606-573f4e6e-ee0e-4d5f-ba8d-a0d5ca04d34c.png)

**Comment by nelsonic** [comment]
Included in PR: #46
